src/editor/deserialize.js

```diff
@@ -37,8 +37,8 @@ function parseHtmlMessage(html) {
                         const resourceId = pillMatch[1]; // The room/user ID
                         const prefix = pillMatch[2]; // The first character of prefix
                         switch (prefix) {
-                            case "@": return new UserPillPart(resourceId, n.textContent);
-                            case "#": return new RoomPillPart(resourceId, n.textContent);
+                            case "@": return new UserPillPart(resourceId, n.textContent, room.getMember(resourceId));
```

Comment:
Can `room` ever be null here?

Reply:
No, it's the current room. If you can write events (to be able to edit them later), the current room should always be an instance of `Room` AFAIK.